alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/util/AlertingUtilsTests.kt

```diff
@@ -0,0 +1,58 @@
+package com.amazon.opendistroforelasticsearch.alerting.util
+
+import com.amazon.opendistroforelasticsearch.alerting.destination.message.BaseMessage
+import com.amazon.opendistroforelasticsearch.alerting.destination.message.CustomWebhookMessage
+import org.elasticsearch.test.ESTestCase
+import java.util.HashMap
+
+class AlertingUtilsTests : ESTestCase() {
+
+    private val denylistRanges = listOf(
```

Comment:
Can call this `DENY_LIST_RANGES` if we want to treat it as a const equivalent for this test class.

Reply:
Changed.